### DISCUSSION TITLE
Loss of focus when editing field value fails to save

### DISCUSSION THREAD

**Discussion by jdshkolnik** [community/community#4931] [Projects and Issues] [answered]
If I'm editing a value and then change focus, the value is lost. I'd expect it to be saved considering the user experience but, at a minimum, it should warn the changes will be lost. Editing the view name does save the value on blur.

**Comment by azenMatt** [comment, marked as answer]
We just updated this so these edits will saved when focus is moved rather than needed an explicit 'enter' confirmation
